Dockerfile

```diff
@@ -6,4 +6,5 @@ WORKDIR $PROJECT_ROOT
 COPY requirements.txt requirements.txt
 RUN pip install -r requirements.txt
 COPY . .
+RUN sed -i -e 's/{{ label }}/Submit/g' /usr/local/lib/python3.6/site-packages/govuk_forms/templates/govuk_forms/submit-button.html
```

Comment:
Unless there's something very strange about the `govuk_forms` package, I don't think we should be modifying anything in `site-packages` - the usual model with Django would be to have a project-specific version of this template within the project's templates directory and [override the default](https://docs.djangoproject.com/en/2.0/howto/overriding-templates/).

Reply:
Also things will probably be simpler if `WORKDIR` is set to `$PROJECT_ROOT/django_reference_app` - that way all the calls to `manage.py` can just be `python3 manage.py subcommand` throughout.